src/test/java/reposense/template/GitTestTemplate.java

```diff
@@ -46,6 +46,7 @@ public class GitTestTemplate {
     protected static final String TEST_COMMIT_HASH_PARENT = "c5a6dc774e22099cd9ddeb0faff1e75f9cf4f151";
     protected static final String MAIN_AUTHOR_NAME = "harryggg";
     protected static final String FAKE_AUTHOR_NAME = "fakeAuthor";
+    protected static final String UNCONVENTIONAL_AUTHOR_NAME = "-invalidGitUsername_TreatedAsUnknownUser";
```

Comment:
Thanks for the contribution. May I propose changing -invalidGitUsername_TreatedAsUnknownUser to another author name value such as -unconventionalAuthorName since the former is no longer invalid or treated as an unknown author with the new regex. Otherwise, this may cause confusion

Reply:
Hi, I believe I didn't change it yet because the original value is used in `annotationTest.java` from the test repository. I can clone the test repository and make a PR there, but could I ask how I could run `gradlew test` using my cloned local version of the test repository?

Reply:
I think it is sufficient to run gradlew in the main repository. The test repositories are for CI dashboard preview